### PR TITLE
Create basic flow around scientific work

### PR DESCRIPTION
**Pull request by JustD4nTe**
1. At start every scientific work have status WaitingForDrawOfReviewers
2. When reviewers got assign to works, status is changed to UnderReview
3. When all reviewers of the work added their reviews, there are 3 options based on avg rating
a) Scientific work is rejected 
b) Author needs to make some corrections to the work
c) Scientific work is approved

Moreover, server sends some emails :P 

Until @upsu90 doesn't make own PR with new view of scientific work, there is nothing to test actually 
You can check status in db and emails
=> after adding work 
=> randomize reviewers 
=> adding all reviews 
=> adding new version 
=> reject/accept work via reviews

### CODE REVIEW

**Review by Kwolik — APPROVED** [review]

File: Kongres.Api/Kongres.Api.Application/Services/EmailSender.cs [hunk 1]
```diff
@@ -8,17 +8,87 @@ public class EmailSender : IEmailSender
...
+        {
+            var message = $"<a href='{WorkLink}{workId}'>Your work has been reviewed!</a>";
+            await _emailService.SendAsync(authorEmail, "Work review", message, true);
+        }
+
+        // When all reviews are received and author can add new version of his work
+        public async Task SendNewVersionEnabledEmailAsync(string authorEmail, uint workId)
+        {
+            var message = $"<a href='{WorkLink}{workId}'>Your work has been rated by all reviewers. You can add a new version now!</a>";
+
```
Comment (upsu90) [inline Kongres.Api/Kongres.Api.Application/Services/EmailSender.cs]: I'll change it to something like that "Your work has been rated by all reviewers and they have some suggestions. You can add a new version now! `<a>Go to work page</a>`"

File: Kongres.Api/Kongres.Api.Application/Services/EmailSender.cs [hunk 2]
```diff
@@ -8,17 +8,87 @@ public class EmailSender : IEmailSender
...
+            await _emailService.SendAsync(authorEmail, "Work review", message, true);
+        }
+
+        public async Task SendToAuthorWorkGotAcceptedAsync(string authorEmail, uint workId)
+        {
+            var message = $"<a href='{WorkLink}{workId}'>Congratulations! Your scientific work is accepted now</a>";
```
Comment (upsu90) [inline Kongres.Api/Kongres.Api.Application/Services/EmailSender.cs]: Maybe a little change?
Congratulations! Your scientific work has just been accepted! `<a> See reviews </a>`

File: Kongres.Api/Kongres.Api.Application/Services/EmailSender.cs [hunk 3]
```diff
@@ -8,17 +8,87 @@ public class EmailSender : IEmailSender
...
+
         public EmailSender(IEmailService emailService)
         {
             _emailService = emailService;
         }
 
+        #region Reviewer        
+        public async Task SendDoNotGetAssignToAnyWork(string reviewerEmail)
+        {
+            const string message = "Unfortunately, you were not assigned to any scientific work, because of too few applicants";
```
Comment (upsu90) [inline Kongres.Api/Kongres.Api.Application/Services/EmailSender.cs]: and we can add:
> Thank you for your commitment! You can still see scientific works. `<a>Go to scientific works page</a>`

File: Kongres.Api/Kongres.Api.Application/Services/EmailSender.cs [hunk 4]
```diff
@@ -8,17 +8,87 @@ public class EmailSender : IEmailSender
...
+            await _emailService.SendAsync(reviewerEmail, "Work review", message);
+        }
+
+        public async Task SendWorkAssignmentInformationAsync(string reviewerEmail, uint workId)
+        {
+            var message = $"<a href='{WorkLink}{workId}'>You have been assigned to scientific work! Go check it out</a>";
```
Comment (upsu90) [inline Kongres.Api/Kongres.Api.Application/Services/EmailSender.cs]: I would just leave it:
> You have been assigned to scientific work! `<a>Go check it out</a>`

**Review by upsu90 — CHANGES_REQUESTED** [review]
Please change the content of emails (links shouldn't be the entire text in mails)

**Review by KamilDonda — APPROVED** [review]

**Comment by KamilDonda** [comment]
+ When reviewers are added to works, the works are still "WaitingForDrawOfReviewers".
+ Scientific works are to be rejected if there are too few reviewers.

**Review by upsu90 — APPROVED** [review]

**Pull request merged**